src/controller/src/clusters.rs

```diff
@@ -639,6 +639,8 @@ where
                             format!("--internal-http-listen-addr={}", assigned["internal-http"]),
                             format!("--opentelemetry-resource=cluster_id={}", cluster_id),
                             format!("--opentelemetry-resource=replica_id={}", replica_id),
+                            format!("--sentry-tag=cluster_id={}", cluster_id),
+                            format!("--sentry-tag=replica_id={}", replica_id),
```

Comment:
Wait, why do we have to specify this as a Sentry tag, given that it's already an OTel resource?

Reply:
so I can deprecate the dual-usage of `--opentelemetry-resource`

Reply:
see below, we can avoid this